Jenkinsfile

```diff
@@ -95,36 +95,38 @@ def rpm_test_daos_test = '''me=\\\$(whoami)
                             trap 'set -x; kill -INT \\\$AGENT_PID \\\$COPROC_PID' EXIT
                             OFI_INTERFACE=eth0 daos_test -m'''
 
-def get_lmd_script = '''set -ex
-                        lmd_tarball='maldetect-current.tar.gz'
-                        lmd_src='maldet-current'
-                        rm -rf ./${lmd_src}
-                        mkdir -p ${lmd_src}
-                        if test -e "${lmd_tarball}"; then
-                          zflag="-z ${lmd_tarball}"
-                        else
-                          zflag=
-                        fi
-                        curl http://rfxn.com/downloads/${tarball} \
-                          ${zflag} --silent --show-error --fail \
-                          -o ${tarball}
-                        tar -C ${lmd_src} --strip-components=1 \
-                          xf ${lmd_tarball}'''
+def rpm_scan_pre = '''set -ex
+                      lmd_tarball='maldetect-current.tar.gz'
+                      if test -e "${lmd_tarball}"; then
+                        zflag="-z ${lmd_tarball}"
+                      else
+                        zflag=
+                      fi
+                      curl http://rfxn.com/downloads/${lmd_tarball} \
+                        ${zflag} --silent --show-error --fail -o ${lmd_tarball}
+                      nodelist=(${NODELIST//,/ })
+                      scp -i ci_key ${lmd_tarball} jenkins@${nodelist[0]}:/tmp
+                      ssh -i ci_key jenkins@${nodelist[0]} "set -ex\n'''
 
-def rpm_scan_daos_test = '''lmd_src=\"${WORKSPACE}/maldet-current\"
-                            pushd ${lmd_src}
-                              sudo ./install.sh
-                              sudo ln -s /usr/local/maldetect/ /bin/maldet
-                            popd
-                            sudo freshclam
-                            rm -f /tmp/clamscan.out
-                            clamscan -d /usr/local/maldetect/sigs/rfxn.ndb \
-                              -d /usr/local/maldetect/sigs/rfxn.hdb -r \
-                              --exclude-dir=/usr/local/maldetect \
-                              --exclude-dir=/usr/share/clamav \
-                              --infected /etc /usr | \
-                              tee /tmp/clamscan.out
-                            grep 'Infected files: 0$' /tmp/clamscan.out'''
+def rpm_scan_test = '''lmd_src=\\\"maldet-current\\\"
+                       lmd_tarball=\\\"maldetect-current.tar.gz\\\"
+                       rm -rf /tmp/\\\${lmd_src}
```

Comment:
`/tmp` is probably not an ideal place for this.  `/var/tmp/` if you really want to just plot it into a `$TMPDIR`, but wouldn't a _workspace_ be the best?

Reply:
No workspace defined on the test vm.  /tmp and /var/tmp are on the same partition of the VM, and this is all discarded since the VM has a snapshot restore on the next use.
It does not look like the $TMPDIR environment environment variable is defined in the quick test I did on a VM.

Reply:
Changed.